lms/services/organization.py

```diff
@@ -82,7 +64,16 @@ def _search_query(self, id_=None, public_id=None, name=None):
                 )
             )
 
-        query = self._db_session.query(Organization)
+        if guid:
+            query = query.join(ApplicationInstance).outerjoin(GroupInfo)
+
+            clauses.extend(
+                sa.or_(
+                    ApplicationInstance.tool_consumer_instance_guid == guid,
+                    GroupInfo.tool_consumer_instance_guid == guid,
+                )
```

Comment:
We _could_ replace this with two separate additions to `clauses` as they are `or`-ed later anyway. But if we want to be able to change the later `or` to an `and` this would still remain an `or`. So I think this is the right formulation.